### PR TITLE
Update VisibleRatioCalculator calculating logic

### PR DESCRIPTION
**Pull request by GeekTree0101**
@jaxtynSong who found visible calculation bug during matrix scroll test.
VisibleRatioCalculator doesn't need to depend on scroll direction while calculating the visible ratio.

https://github.com/daangn/SushiBelt/assets/19504988/a989698b-cff2-4f63-9c35-7ea3b498a02e

### CODE REVIEW

**Comment by GeekTree0101** [comment]
✗ test_early_exit_in_scrollDirection_null, XCTAssertNil failed: "nan"

**Review by ElonPark — APPROVED** [review]
LGTM 👍👍

File: Example/SushiBelt/MatrixViewController.swift [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  }
+}
+
+class MatrixViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate {
+
+  var collectionView: UICollectionView!
+  private let tracker = SushiBeltTracker()
+  private let debugger = SushiBeltDebugger.shared
+  private let sections: [[Sushi]] = [
+    Sushi.いらっしゃいませ(count: 10),
```
Comment (KYHyeon) [inline Example/SushiBelt/MatrixViewController.swift:52]: すごいですね 😂

**Review by jaxtynSong — APPROVED** [review]
👍  THX :)

**Pull request merged**